Makefile

```diff
@@ -1,16 +1,15 @@
-CXX		?= g++
+CXX		= g++-12
 SRCDIR		= ./src
 TESTDIR		= ./tests
 BENCHDIR	= ./benchmarks
 UTILS		= ./utils
 SRCS		= $(wildcard $(SRCDIR)/*.hpp)
 TESTS		= $(wildcard $(TESTDIR)/*.cpp)
 TESTOBJS	= $(patsubst $(TESTDIR)/%.cpp,$(TESTDIR)/%.o,$(TESTS))
-TESTOBJS	:= $(filter-out $(TESTDIR)/main.o ,$(TESTOBJS))
 CXXFLAGS	+= -I$(SRCDIR) -I$(UTILS)
 GTESTCFLAGS	= `pkg-config --cflags gtest`
 GTESTLDFLAGS	= `pkg-config --libs gtest`
```

Comment:
You'll want to reference `gtest_main` to get all the flags in one go.

Reply:
+1